src/test/java/com/checkmarx/flow/cucumber/integration/cxconfigbugtracker/CxConfigBugTrackerSteps.java

```diff
@@ -43,12 +45,12 @@
 @SpringBootTest(classes = {CxFlowApplication.class, CxConfigBugTrackerConfiguration.class, PublishUtils.class})
 @Slf4j
 public class CxConfigBugTrackerSteps {
-    private static final String PULL_REQUEST_STATUSES_URL = "statuses url stub";
     public static final String BRANCH_NAME = "udi-tests";
     public static final String CUSTOM_BEAN_NAME = "GitHub";
     public static final String GITHUB_USER = "cxflowtestuser";
 
     @Autowired
+    @Qualifier("cxConfigurationTestBean")
     private CxClient cxClientMock;
```

Comment:
This can be shortened to something like
```
    @MockBean
    private final CxClient cxClientMock;
```
without the need to include CxConfigBugTrackerConfiguration.